experimental/traffic-portal/src/app/core/certs/cert-author/cert-author.component.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Component, Input } from "@angular/core";
+
+import { Author } from "src/app/core/certs/cert-detail/cert-detail.component";
+
+/**
+ * CertAuthorComponent is the controller used for displaying a cert author.
+ */
+@Component({
+	selector: "tp-cert-author",
+	styleUrls: ["./cert-author.component.scss"],
+	templateUrl: "./cert-author.component.html"
+})
+export class CertAuthorComponent {
+	@Input() public author: Author | undefined;
```

Comment:
I don't think this component serves any purpose or function without an author to display - why make it optional?